docs/source/markdown/podman-manifest-push.1.md.in

```diff
@@ -55,6 +55,10 @@ Sign the pushed images with a “simple signing” signature using the specified
 
 Sign the pushed images with a sigstore signature using a private key at the specified path. (This option is not available with the remote Podman client, including Mac and Windows (excluding WSL2) machines)
 
+#### **--sign-passphrase**=*passphrase*
+
+If signing the image with **--sign-by**, set the signing passphrase
```

Comment:
Add conflicts with --sign-passphraze-file.

And add conflicts with --sign-passphrase below